src/Morphic-Widgets-Scrolling/ScrollPane.class.st

```diff
@@ -733,7 +733,7 @@ ScrollPane >> scrollBy: delta [
 		ifTrue: [scrollBar value: 0.0]
 		ifFalse: [scrollBar value: newYoffset asFloat / r].
 	(r := self hLeftoverScrollRange) = 0
-		ifTrue: [hScrollBar value: -3.0]
+		ifTrue: [hScrollBar value: (self hMargin negated asFloat)]
```

Comment:
Is this useful? I see in ScrollBar>value: that the value should be between 0.0 and 1.0. (There is a #min:max: to ensure it)

Reply:
I think you're right. It seems the original code was in error and I changed it to the margin without considering that.